test/Polly.Core.Tests/CircuitBreaker/IsolatedCircuitExceptionTests.cs

```diff
@@ -7,10 +7,16 @@ public class IsolatedCircuitExceptionTests
     [Fact]
     public void Ctor_Ok()
     {
-        new IsolatedCircuitException("Dummy.").Message.Should().Be("Dummy.");
-        new IsolatedCircuitException().Message.Should().Be("The circuit is manually held open and is not allowing calls.");
-        new IsolatedCircuitException("Dummy.", new InvalidOperationException()).Message.Should().Be("Dummy.");
-        new IsolatedCircuitException("Dummy.", new InvalidOperationException()).InnerException.Should().BeOfType<InvalidOperationException>();
+        IsolatedCircuitException dummyMessageException = new("Dummy.");
+        dummyMessageException.Message.Should().Be("Dummy.");
+        dummyMessageException.RetryAfter.Should().BeNull();
+        IsolatedCircuitException defaultException = new();
+        defaultException.Message.Should().Be("The circuit is manually held open and is not allowing calls.");
+        defaultException.RetryAfter.Should().BeNull();
+        IsolatedCircuitException dummyMessageExceptionWithInnerException = new("Dummy.", new InvalidOperationException());
```

Comment:
A question of style, but I prefer the `var foo = new Bar()` style over `Bar foo = new()` for variables.